spec/models/date_range_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+RSpec.describe DateRange do
+  
+  describe '#compact' do
+    let(:start_date) { Date.parse('2015-12-31') }
+    let(:end_date)   { Date.parse('2015-12-31') }
+    subject { described_class.new(start_date: start_date, end_date: end_date).display }
```

Comment:
Leaving a blank line between the `let` statements and the subject would be 💚 - makes things way better to read.